src/recovery/store/sqlite.rs

```diff
@@ -222,11 +202,10 @@ impl KWriter<StoreKey, Change<StateBytes>> for SqliteStateWriter {
                     Change::Discard => None,
                 };
                 let sql = format!(
-                    "INSERT INTO {} (step_id, state_key, epoch, snapshot) \
+                    "INSERT INTO state (step_id, state_key, epoch, snapshot) \
                      VALUES (?1, ?2, ?3, ?4) \
                      ON CONFLICT (step_id, state_key, epoch) DO UPDATE \
                      SET snapshot = EXCLUDED.snapshot",
-                    self.table_name,
                 );
                 let future = query(&sql)
```

Comment:
Since we are not building the string dinamically anymore, you could skip allocating a new string:

```rust
let sql = "INSERT INTO ...."
let future = query(sql)....
```